Common/include/option_structure.hpp

```diff
@@ -929,6 +929,34 @@ static const MapType<std::string, TURB_MODEL> Turb_Model_Map = {
   MakePair("SST_SUST", TURB_MODEL::SST_SUST)
 };
 
+/*!
+ * \brief Families of turbulence models
+ */
+enum class TURB_FAMILY {
+  NONE,   /*!< \brief No turbulence model. */
+  SA,     /*!< \brief Spalart-Allmaras variants. */
+  KW,     /*!< \brief k-w models. */
+};
+/*!
+ * \brief Associate turb models with their family
+ */
+inline TURB_FAMILY TurbModelFamily(TURB_MODEL model) {
+  switch (model) {
+    case TURB_MODEL::NONE:
+      return TURB_FAMILY::NONE;
+    case TURB_MODEL::SA:
+    case TURB_MODEL::SA_NEG:
+    case TURB_MODEL::SA_E:
+    case TURB_MODEL::SA_COMP:
+    case TURB_MODEL::SA_E_COMP:
+      return TURB_FAMILY::SA;
+    case TURB_MODEL::SST:
+    case TURB_MODEL::SST_SUST:
+      return TURB_FAMILY::KW;
+  }
+  return TURB_FAMILY::NONE;
```

Comment:
What do you need this additional return for? In case a new TURB_MODEL (e.g. k-eps) is added the `enum class` forces the person who writes the turb model to handle this ... of course this extra catch helps against a lazy `default: break;` fix. In case there is another reason let me know

Reply:
Compiler warning otherwise "control may reach the end of function with return type" (loosely paraphrasing)

Reply:
Just to reassure my understanding here: The compiler warning is "wrong" in this special case with what i state above - without a `default break` the switch of an `enum class`, where each option is followed by a return, cannot end without a return. But I guess that is why its a just warning ...